integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/TransferDashFragment.kt

```diff
@@ -211,15 +211,17 @@ class TransferDashFragment : Fragment(R.layout.transfer_dash_fragment) {
         }
 
         transferDashViewModel.observeCoinbaseAddressState.observe(viewLifecycleOwner){ address ->
-            val amountFiat = if (fiatValue != null) dashFormat.format(fiatValue).toString() else getString(R.string.transaction_row_rate_not_available)
-            val fiatSymbol = if (fiatValue != null) GenericUtils.currencySymbol(fiatValue?.currencyCode) else ""
-            val amountStr = dashValue.toPlainString()
-            val fee = VALUE_ZERO
-            val total = dashValue.toPlainString()
+            val fiatVal = enterAmountToTransferViewModel.getFiat(dashValue.toPlainString())
+            val amountFiat = dashFormat.format(fiatVal).toString()
+            val fiatSymbol = GenericUtils.currencySymbol(fiatVal.currencyCode)
+
             lifecycleScope.launch {
+                val details = transactionDetails.estimateNetworkFee(transferDashViewModel.dashAddress, dashValue)
```

Comment:
I wonder if we should do this in CrowdNode to estimate fee for the topup. @HashEngineering you think it would make sense, or just leave it a constant?

Reply:
I was wondering something similar, but regarding the `sendCoins` methods in the `SendCoinsTaskRunner` class.  

Regarding the Crowdnode and estimating fees, this method may be useful.  It may help prevent very rare edge case problems. though such problems may not be possible.